lib/osml/model_runner/roles/mr_task_role.ts

```diff
@@ -195,7 +195,13 @@ export class MRTaskRole extends Construct {
         "logs:DescribeLogStreams",
         "logs:DescribeLogGroups",
         "logs:CreateLogStream",
-        "logs:CreateLogGroup"
+        "logs:CreateLogGroup",
+        "logs:StartQuery",
```

Comment:
I don't think these permissions are necessary. Why would the ModelRunner ECS task need to run CloudWatch queries?